mathics/builtin/image/misc.py

```diff
@@ -69,11 +69,24 @@ class ImageImport(Builtin):
      = -Image-
     """
 
+    messages = {
+        "imgtype": "Can't identify image file type for `1`.",
```

Comment:
@mmatera I have no idea what Mathematica does in this situation. Please feel free to suggest or adjust the error messages.

Reply:
In WMA, there is not a symbol `System`ImageImport`. If I fill a file `a.tif` with random bytes, using `Import`, in WMA I get


```
In[1]:= Import["a.tif"]                                                         

Import::infer: Cannot infer format of file a.tif.

Out[1]= $Failed
```

Reply:
@mmatera  thanks - 7a2567ce0 has this message adjusted. When CI clear I will merge that the `replace-lena` branch.